src/Controller/NewsController.php

```diff
@@ -89,8 +89,9 @@ public function show($slug, Request $request)
         try {
             $page = $this->api->getPageByUrl($request->getRequestUri());
             $response = HttpClient::create()->request('GET', getenv('APP_API_BASE_URL') . 'wp/v2/posts/' . $page->getId());
-
-            if ($response->getStatusCode() == 200) {
+            // check if 'author_name_text' acf is set for that page in wp
+            $authorDefined = property_exists(json_decode($response->getContent())->acf, 'author_name_text');
+            if ($response->getStatusCode() == 200 && $authorDefined) {
```

Comment:
we should check **author_name_text** once we have the 200 responses.
`if ($response->getStatusCode() == 200) {
                $acfContent = (array) json_decode($response->getContent())->acf;
                $authorName =  array_key_exists('author_name_text', $acfContent) ? $acfContent["author_name_text"] : null;
}`